components/app-sidebar.tsx

```
@@ -56,55 +70,88 @@ const data = {
             </SidebarHeader>
-            <SidebarContent>
-                <NavMain items={data.navMain} />
-                <NavProjects projects={data.utility} />
+            <SidebarContent className="flex flex-col gap-y-1 px-2">
+                <div className="px-1 py-2">
+                    <Dialog open={!!tradeComposerOpen} onOpenChange={handleOpenChange}>
+                        <DialogTitle></DialogTitle>
+                        <DialogDescription></DialogDescription>
+
+                        <DialogTrigger asChild>
+                            <button className="inline-flex h-8 w-full items-center justify-center gap-1 self-stretch overflow-hidden rounded-md border border-gray-200 bg-transparent text-black dark:border-none dark:bg-gradient-to-b dark:from-white/20 dark:to-white/10 dark:text-white dark:outline dark:outline-1 dark:outline-offset-[-1px] dark:outline-white/5">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)